tpie/sort_manager.h

```diff
@@ -828,4 +828,4 @@ void sort_manager<T,I,M>::single_merge(
 
 }  //  tpie namespace
 
-#endif // _TPIE_AMI_SORT_MANAGER_H 
+#endif // _TPIE_AMI_SORT_MANAGER_H
```

Comment:
Puh, there is a lot of whitespace noise here. This will mess up blame, and makes it harder to see whats atcually changes. On the other hand most of it does make sense...